docs/UserGuide.md

```diff
@@ -287,7 +294,7 @@ Format: `delete-wedding w/WEDDINGNAME [f/]`
 * The no contacts should be assigned to the `Wedding` before it is deleted.
```

Comment:
Typo/ grammar error here - "There should not be contacts assigned to.."/"There should be no"?